galleries/views.py

```diff
@@ -1,6 +1,5 @@
 from django.shortcuts import render,get_object_or_404
 from .models import Gallery, Tag
-from django.utils.text import slugify
 
 # Create your views here.
 def galleries(request):
```

Comment:
I would avoid fetching objects.all() in the code. Best practise is to page your data (fetching only a 'page' of items, e.g. 50 each)